cruise-control-client/cruisecontrolclient/client/Endpoint.py

```diff
@@ -361,6 +361,19 @@ class PauseSamplingEndpoint(AbstractEndpoint):
         'kwargs': dict(aliases=[name.replace('_', '-')], help=description)
     }
 
+class PermissionsEndpoint(AbstractEndpoint):
```

Comment:
I would recommend to use "GetPermissionEndpoint" as the endpoint name. "PermissionsEndpoint" sounds like it can update and view the permission, while it actually does not change permissions. I'd also recommend changing the PR name to Add GetPermission endpoint API. Let me know if it makes sense.

Reply:
Ack

Reply:
We can rename the class to "GetPermissionsEndpoint", but I would leave the REST endpoint path name as "../permissions" to follow the REST API naming conventions: 
https://medium.com/@nadinCodeHat/rest-api-naming-conventions-and-best-practices-1c4e781eb6a5